O21.Game/Localization/Translations.fs

```diff
@@ -0,0 +1,16 @@
+module O21.Localization.Translations
+
+open FSharp.Data
+
+type Language = 
+    | English
+    | Russian
+
+type private Provider = JsonProvider<"Localization\Translations\english.json">
```

Comment:
This path looks like Windows-specific, or maybe just outright broken. Could you please take a look and make it portable?

Reply:
I think adding Path.Combine would be ok

Reply:
Nope. This is getting calculated in constant context, so you cannot call runtime functions here. But a replacement `\` with `/` should fix the issue.